gateway.c

```diff
@@ -604,6 +606,8 @@ void LoadConfigFile()
 	ReadString(fp, "ftpPassword", Config.ftpPassword, sizeof(Config.ftpPassword), 0);
 	ReadString(fp, "ftpFolder", Config.ftpFolder, sizeof(Config.ftpFolder), 0);	
 
+	ReadBoolean(fp, "EnableKML", 0, &Config.EnableKML);

```

Comment:
Don`t make this a boolean. I would like to add different log levels, to record full telemetry and/or KML and/or RSSI / frequency offset etc. Perhaps change the name to "LogLevel" and use a bitmask. Thanks.